.zenodo.json

```diff
@@ -98,6 +98,11 @@
         "name": "Gu, Junmin",
         "orcid": "0000-0002-1521-8534"
       },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Haseeb, Muhammad",
+        "orcid": ""
```

Comment:
@mhaseeb123 Could you let us know what your ORCID number is?

Reply:
https://orcid.org/0000-0002-0697-6894